src/config/config.ts

```diff
@@ -1,13 +1,18 @@
 import { TOKENS, Token, UnderlyingCollateralToken } from '../types';
-import { goerliConfig } from './goerli';
-import { mainnetConfig } from './mainnet';
-import { NetworkConfig, SupportedCollateralTokens, SupportedNetwork } from './types';
+import { goerliConfig, goerliRaftTokenConfig } from './goerli';
+import { mainnetConfig, mainnetRaftTokenConfig } from './mainnet';
+import { NetworkConfig, RaftTokenConfig, SupportedCollateralTokens, SupportedNetwork } from './types';
 
 const networkConfig: Record<SupportedNetwork, NetworkConfig> = {
   mainnet: mainnetConfig,
   goerli: goerliConfig,
 };
 
+const raftTokenConfig: Record<SupportedNetwork, RaftTokenConfig> = {
```

Comment:
I think we don't need yet another config. Let's keep it together with the main one since both contain addresses for the same networks.

Reply:
sure if we want to. will do it in next PR